### PR TITLE
[CGPROD-502] Added headers for gameplay component docs

### PR DESCRIPTION
**Pull request by Thomas-James-Rose**
Added headers for gameplay component docs and split out non-shared questions from README.md.

### CODE REVIEW

File: docs/gameplay-component.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* [How is Genie structured?](#how-is-genie-structured)
+* [How do I use the three core engines?](#how-do-i-use-the-three-core-engines)
+* [What coding and test standards do I need to apply?](#what-coding-and-test-standards-do-i-need-to-apply)
+* [How does my component plug into Genie?](#how-does-my-component-plug-into-genie)
+* [How do I test my component in the Genie framework?](#how-do-i-test-my-component-in-the-genie-framework)
+* [Are there any areas of existing Children’s game delivery standards I need to apply?](#are-there-any-areas-of-existing-children’s-game-delivery-standards-i-need-to-apply)
+* [How do I get my build onto Children’s platforms?](#how-do-i-get-my-build-onto-children’s-platforms)
+* [What acceptance tests will the BBC carry out?](#what-acceptance-tests-will-the-bbc-carry-out)
+* [What documentation do I need to supply?](#what-documentation-do-i-need-to-supply)
+* [How can I access support?](#how-can-i-access-support)
```
Comment (Leonie-) [inline docs/gameplay-component.md]: Should this maybe live at the top level, as this question is not specific to the gameplay component?

File: docs/gameplay-component.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Children's BBC - Genie Gameplay Component
+
+* [How is Genie structured?](#how-is-genie-structured)
+* [How do I use the three core engines?](#how-do-i-use-the-three-core-engines)
```
Comment (Leonie-) [inline docs/gameplay-component.md]: I think we are just calling them "core engines" now, and not numbering them.

File: docs/theming.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## How do I test my theme?
+
+
+## What are the applicable Children’s Games Standards that need to be followed?
+
+
+## How do I get my game onto Children’s platforms?
+
+Every time a commit is made to the repository, our Jenkins job will build the game to our Children's Game Embed (CAGE) page. You will get email notifications with status reports on success/failure of any automated builds. In the event of a failure you should get an error report with debug information.
```
Comment (Leonie-) [inline docs/theming.md]: This answer is more applicable to the gameplay component than the theme - maybe it should be moved (as we don't quite know what a theme build will look like yet).

File: docs/gameplay-component.md (hunk 1 above)
Comment (Thomas-James-Rose) [reply docs/gameplay-component.md]: I agree. At the time of writing I wasn't sure if there were separate and specific support processes for creating a gameplay component and making a theme. Regardless, this should probably be at the top level of the repo.

**Comment by Thomas-James-Rose** [comment]
Updated as per review comments.

**Comment by Leonie-** [comment]
Great stuff :pineapple:

**Pull request merged**